pkg/controller/account/account_controller.go

```diff
@@ -528,6 +528,33 @@ func (r *ReconcileAccount) nonCCSAssignAccountID(reqLogger logr.Logger, currentA
 		return err
 	}
 
+	configMap, err := controllerutils.GetOperatorConfigMap(r.Client)
+	if err != nil {
+		reqLogger.Error(err, "failed getting shard name from configmap data")
+		return err
+	}
+
+	hiveName, ok := configMap.Data["shard-name"]
+
+	if ok {
+		// Add tag
+		inputTag := &organizations.TagResourceInput{
+			ResourceId: aws.String(awsAccountID),
+			Tags: []*organizations.Tag{
+				{
+					Key:   aws.String("owner"),
+					Value: aws.String(hiveName),
+				},
+			},
+		}
+
+		_, err = awsSetupClient.TagResource(inputTag)
+		if err != nil {
+			return err
+		}
+
+	}
+
```

Comment:
This entire block of code would probably be better suited as a function on the reconcile object that is called here.

So what I would do would be to take this block and add it to a new function called `r.tagAccount`, so here would be replaced with something like:

```
err = r.tagAccount(awsSetupClient)
if err != nil {
  reqLogger.Info("Unable to tag aws account.", "account", currentAcctInstance.Name, "AWSAccountID", awsAccountID, "Error", error.Error())
}
```

And then tagAccount would be defined further down and would look like this:

```
func (r *ReconcileAccount) tagAccount(awsSetupClient awsClient.Client) error {
  [code above]
}
```